python/ray/experimental/client/common.py

```diff
@@ -228,63 +194,16 @@ def __call__(self, *args, **kwargs):
         raise TypeError(f"Remote method cannot be called directly. "
                         "Use {self._name}.remote() instead")
```

Comment:
```suggestion
                        f"Use {self._name}.remote() instead")
```
otherwise it will print "Use {self._name}.remote() instead"